app/cdap/components/GridTable/index.tsx

```diff
@@ -164,20 +178,24 @@ export default function GridTable() {
   // ------------@getGridTableData Function is used for preparing data for entire grid-table
   const getGridTableData = async () => {
     const rawData: IExecuteAPIResponse = gridData;
-    const headersData = createHeadersData(rawData.headers, rawData.types);
+    const headersData: IHeaderNamesList[] = createHeadersData(
+      rawData.headers,
+      rawData.types
+    ) as IHeaderNamesList[];
     setHeadersNamesList(headersData);
     if (rawData && rawData?.summary && rawData?.summary?.statistics) {
-      const missingData = createMissingData(gridData?.summary?.statistics);
+      const missingData: IMissingListData[] = createMissingData(gridData?.summary?.statistics);
```

Comment:
createMissingData rename to getMissingList
IMissingListData rename to IMissingList
missingData rename to missingItems

rename all 3 variabled to more meaningful name with what data it is?